### PR TITLE
Manual cleanup

### PR DESCRIPTION
**Pull request by CalMacCQ**
Made several small changes to the manual. I plan to update the workflow to have the html pages built on the CI in a future PR. 

**Changes**
- Use more subtle "borland" pygments (suggested by Alec) for code cells instead of "pastie"
- Add sphinx copy button for code cells - there are a few extra js and css files in `_static` for this
- Fix broken links
- Add links to extensions and example notebooks
- Tidy up code cell spacing and comments - hopefully improves readability
- Add Quantinuum logo
- Update to copyright 2023

If this is too many changes for one PR let me know.

Screenshot 2023-01-13 at 19 07 42

### CODE REVIEW

File: docs/manual/_static/copybutton.js [hunk 1]
```diff
@@ -0,0 +1,248 @@
+// Localization support
+const messages = {
+  'en': {
+    'copy': 'Copy',
+    'copy_to_clipboard': 'Copy to clipboard',
+    'copy_success': 'Copied!',
+    'copy_failure': 'Failed to copy',
+  },
+  'es' : {
```
Comment (cqc-melf) [inline docs/manual/_static/copybutton.js:9]: Are we supporting the other languages somewhere else? Do we need this?

File: docs/manual/_sources/index.rst.txt [hunk 2]
```diff
@@ -23,10 +24,12 @@ This manual refers to the following versions of pytket and extensions:
     manual_assertion.rst
 
 .. toctree::
-	:caption: More documentation:
-	:maxdepth: 1
+    :caption: More Documentation:
```
Comment (cqc-melf) [inline docs/manual/_sources/index.rst.txt:27]: Is this working fine with the removed space? Have you checked that?

File: docs/manual/_static/copybutton.js [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+
+let doc_url_root = DOCUMENTATION_OPTIONS.URL_ROOT;
+if (doc_url_root == '#') {
+    doc_url_root = '';
+}
+
+/**
+ * SVG files for our copy buttons
+ */
+let iconCheck = `<svg xmlns="http://www.w3.org/2000/svg" class="icon icon-tabler icon-tabler-check" width="44" height="44" viewBox="0 0 24 24" stroke-width="2" stroke="#22863a" fill="none" stroke-linecap="round" stroke-linejoin="round">
```
Comment (cqc-melf) [inline docs/manual/_static/copybutton.js:61]: Do we want to include external files?

File: manual/index-rst-template [hunk 4]
```diff
@@ -18,10 +19,12 @@ REQUIREMENTS
     manual_assertion.rst
 
 .. toctree::
-	:caption: More documentation:
-	:maxdepth: 1
+    :caption: More Documentation:
```
Comment (cqc-melf) [inline manual/index-rst-template:22]: See above

File: docs/manual/_static/copybutton.js (hunk 3 above)
Comment (CalMacCQ) [reply docs/manual/_static/copybutton.js:61]: This file is for the sphinx-copybutton I added which allows the code cells to be easily copied. If I'm not mistaken we need to add this file (at least this first time) in order for it to work.

Do you think this is okay?

File: docs/manual/_static/copybutton.js (hunk 1 above)
Comment (CalMacCQ) [reply docs/manual/_static/copybutton.js:9]: Thanks, see below.

File: docs/manual/_sources/index.rst.txt (hunk 2 above)
Comment (CalMacCQ) [reply docs/manual/_sources/index.rst.txt:27]: Yes, I double checked that it builds correctly. Thanks.

**Review by cqc-melf — APPROVED** [review]

**Pull request merged**